cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/dto/OrderInputDTO.java

```diff
@@ -60,6 +60,7 @@
     @ApiModelProperty( "When in reseller mode, specify the reseller username" )
     private String reseller;
 
+    @ApiModelProperty( "Operating system for a bare VM" )
```

Comment:
this will override the description inside OsType or any other description for the same type in a different class